lib/rdoc/markup/to_html.rb

```diff
@@ -458,4 +462,104 @@ def to_html(item)
     super convert_flow @am.flow item
   end
 
+  private
+
+  def append_to_tidy_label(fragment)
+    return unless @tidy_link_buffer
+
```

Comment:
It seems that we can remove this check here because it seems that callers are responsible to check `tidy_link_capturing?` before calling this.